### DISCUSSION TITLE
Is Chrome Web Store being updated with 2.16.0?

### DISCUSSION THREAD

**Discussion by max5604** [violentmonkey/violentmonkey#1946] [General]
I see there is a new 2.16.0 GitHub Release, but it looks like Chrome Web Store still has 2.15.10 on https://chrome.google.com/webstore/detail/violentmonkey-beta/opokoaglpekkimldnlggpoagmjegichg

**Comment by tophf** [comment]
No, but the code is 100% identical.

Also, you've linked **beta**, whereas the [stable release](https://chromewebstore.google.com/detail/violentmonkey/jinjaccalgkegednnccohejagnlnfdag) is still being reviewed, which is slow.
